src/ui/resource/Image.js

```diff
@@ -147,7 +148,9 @@ exports = Class(lib.PubSub, function () {
     // create an image if we don't have one
     if (!img) {
       img = new Image();
-      img.crossOrigin = 'use-credentials';
+      if (!this._cors_enabled) {
+        img.crossOrigin = 'anonymous';
```

Comment:
if you set the value from opts.cors, the caller will have the freedom to set it to whatever they want.
Right now use-credentials is never being set.

Reply:
Updated for that.